utils/frame/try-runtime/cli/Cargo.toml

```
@@ -0,0 +1,32 @@
+[package]
+name = "try-runtime-cli"
+version = "3.0.0"
```

```suggestion
version = "0.8.0"
```

this should be considered beta for now